actionpack/test/controller/render_test.rb

```diff
@@ -368,6 +379,18 @@ def test_expires_now_with_cache_control_headers
     assert_match(/no-transform/, @response.headers["Cache-Control"])
   end
 
+  def test_expires_now_with_conflicting_cache_control_headers
+    get :conditional_hello_with_expires_and_confliciting_cache_control_headers
+    assert_match(/no-cache/, @response.headers["Cache-Control"])
+    refute_match(/must-revalidate/, @response.headers["Cache-Control"])
```

Comment:
I kind of hate `refute_match` since it can pass for any number of reasons. I wonder if just doing a string compare on the `Cache-Control` header would be safest in this test.